app/renderer/js/main.js

```diff
@@ -625,6 +626,22 @@ class ServerManagerView {
 			});
 		});
 
+		ipcRenderer.on('toggle-autohide-menubar', (event, autoHideMenubar, updateMenu) => {
+			if (updateMenu) {
+				ipcRenderer.send('update-menu', {
+					tabs: this.tabsForIpc,
+					activeTabIndex: this.activeTabIndex
+				});
+				return;
+			}
+			const selector = 'webview:not([class*=disabled])';
```

Comment:
@kanishk98 can you make a function for this since we are using this in `toggle-sidebar` also?